backend/x11/backend.c

```diff
@@ -15,8 +15,8 @@
 #include <wlr/render/gles2.h>
 #include <wlr/util/log.h>
 #include <X11/Xlib-xcb.h>
-#include <xcb/glx.h>
 #include <xcb/xcb.h>
+#include <xcb/xcb_event.h>
```

Comment:
I've been reading some of your code and found the following in `xwayland/xwm.c`:
```
/* This is in xcb/xcb_event.h, but pulling xcb-util just for a constant
 * others redefine anyway is meh
 */
#define XCB_EVENT_RESPONSE_TYPE_MASK (0x7f)
```

Reply:
For some reason, I thought we were pulling in xcb-util, but I guess I was mistaken.
Yeah, it's really not worth it just for that.

Reply:
Right, I had forgotten I did that myself, we need to either add the dependency to meson or do the same